frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -16,8 +16,7 @@ const GEO_DATASET = "GEO_DATASET";
 const GEO_ID = "id";
 
 const VAR_DATASET = "VAR_DATASET";
-const VAR_STATE_FIPS = "state_fips";
-const VAR_COUNTY_FIPS = "COUNTY_FIPS";
+const VAR_FIPS = "fips";
```

Comment:
I wonder if we should make a file of constants with standard column names? Not blocking this PR, just a thought to make refactoring easier because right now it's tricky to search for all usages of a particular column name

Reply:
added a TODO :)